controls.py

```diff
@@ -5,64 +5,57 @@
 
 
 ######################################Creation Functions
-def create_student(username, name, password, email, token=""):
+def create_account(username):
     # Create a Student and Account instance in the rd database.
     s = db.get_session()
 
-    Acc =  Account(username=username, name=name, password=password, email=email, account_token=token)
+    Acc =  Account(username)
     s.add(Acc)
-    s.flush()
-
-    Stu = Student(related_account=Acc.id)
-    s.add(Stu)
     s.commit()
     s.close()
+
     print(username, "added to the database")
 
-def create_donor(username, password, email, token=""):
-    # Create a Donor and Account instance in the rd database
+def approve_admin(usernameAttempt):
     s = db.get_session()

```

Comment:
I think we also have an ADMIN dictionary in config.py, maybe we should have it so admins are added to that list in config as well?